### PR TITLE
PLAP-503 Added openapi3 docs for email-verification-service

### PR DESCRIPTION
**Pull request by nstergiou-prodyna**

### CODE REVIEW

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+openapi: 3.0.1
+info:
+  title: Email Verification Service
+  description: Service that performs the email verification of a user
+  contact: {}
+  version: "1.0"
+servers:
+- url: /
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:8]: Hi @nstergiou-prodyna !

Please add the server urls for our APIs:

```yaml
servers:
  - url: https://services.dev.lionaire.app/oneapp-email-verification-service
    description: Placons Development environment
  - url: https://services.stg.lionaire.app/oneapp-email-verification-service
    description: Placons Staging environment
  - url: https://services.prd.lionaire.app/oneapp-email-verification-service
```
Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/app.VerificationEmail'
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:27]: Hi @nstergiou-prodyna !

Please remove the `app` prefix which was added automatically by `swag` using the go package:

```diff
-$ref: '#/components/schemas/app.VerificationEmail'
+$ref: '#/components/schemas/VerificationEmail'

Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    post:
+      summary: Post verification code in order to verify user's email
+      description: Verifies the code the user send after receiving the email to verify
+        his email address
+      requestBody:
+        description: Verify Code
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ports.EmailVerificationCodeRequest'
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:52]: Hi @nstergiou-prodyna !

Please remove the `ports.` prefix which was added automatically by `swag` using the go package:

```diff
-$ref: '#/components/schemas/ports.EmailVerificationCodeRequest'
+$ref: '#/components/schemas/EmailVerificationCodeRequest'

Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    post:
+      summary: Post email verification request
+      description: Sends the verification email to the user. The email contains a
+        verification code
+      requestBody:
+        description: Send Verification Email
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ports.EmailVerificationRequest'
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:69]: Hi @nstergiou-prodyna !

Please remove the `ports.` prefix which was added automatically by `swag` using the go package:

```diff
-$ref: '#/components/schemas/ports.EmailVerificationRequest'
+$ref: '#/components/schemas/EmailVerificationRequest'

Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        required: true
+      responses:
+        201:
+          description: CREATED
+          content: {}
+      x-codegen-request-body-name: request
+components:
+  schemas:
+    app.VerificationEmail:
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:78]: Hi @nstergiou-prodyna !

Please remove the `app` prefix which was added automatically by `swag` using the go package:

```diff
-app.VerificationEmail:
+VerificationEmail:

Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+      type: object
+      properties:
+        email:
+          type: string
+        validUntilDate:
+          type: string
+        verified:
+          type: boolean
+    ports.EmailVerificationCodeRequest:
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:87]: Hi @nstergiou-prodyna !

Please remove the `ports.` prefix which was added automatically by `swag` using the go package:

```diff
-ports.EmailVerificationCodeRequest:
+EmailVerificationCodeRequest:
```
Thanks!

File: openapi-docs/oneapp-email-verification-service.yaml [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+      type: object
+      properties:
+        code:
+          type: string
+        email:
+          type: string
+    ports.EmailVerificationRequest:
```
Comment (cmanios) [inline openapi-docs/oneapp-email-verification-service.yaml:94]: Hi @nstergiou-prodyna !

Please remove the `ports.` prefix which was added automatically by `swag` using the go package:

```diff
-ports.EmailVerificationRequest:
+EmailVerificationRequest:
```

Thanks!

**Review by cmanios — CHANGES_REQUESTED** [review]

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 1 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml:8]: I wouldn't want to add anything extra, let's keep this as an "as-is migration" task.
Some of the services are not publicly available anyway

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 2 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 3 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 5 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 6 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 7 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml:94]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 4 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Done

File: openapi-docs/oneapp-email-verification-service.yaml (hunk 1 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-email-verification-service.yaml]: Added dev, stg urls as agreed

**Review by cmanios — APPROVED** [review]

**Pull request merged**